src/components/LandingPage/HeroSection/index.tsx

```diff
@@ -116,7 +116,7 @@ function Index() {
           <RotatedHeadline
             interval={2000}
             lines={[
-              "Build in Cypherspace",
+              "Internet is the stack",
```

Comment:
```suggestion
              "Internet is the stack", 
```
silly change to trigger CI again